### DISCUSSION TITLE
Propose see-through labels.

### DISCUSSION THREAD

**Discussion by IneovaAI** [ultralytics/yolov5#4335] [Ideas]
![Compare_Labels](https://user-images.githubusercontent.com/67843470/128600539-6e47997b-e15c-46fb-b306-e33ceee8b9dd.jpg)

Good day. I have been working on a script to make the labels see-through and just overall better looking in my opinion. Attached is an image that compares the two. If anyone is interested in the code please let me know and I will make the code available. It is still a work in progress tho.

**Comment by glenn-jocher** [comment]
@IneovaAI interesting. I use alpha as a function of confidence in our iOS app.

**Reply by glenn-jocher** [reply]
We have a few improvements to the app on the way also :)

![Screenshot 2021-08-05 at 20 19 37](https://user-images.githubusercontent.com/26833433/129050839-1b71baf5-4876-4fc4-8c8c-e6c4d07021db.jpg)
